src/openms/include/OpenMS/METADATA/ID/MetaData.h

```diff
@@ -64,20 +66,14 @@ namespace OpenMS
     {
       PROTEIN,
       COMPOUND,
-      RNA,
-      SIZE_OF_MOLECULETYPE
+      RNA
```

Comment:
is the SIZE_OF_... entry not needed anymore?

Reply:
I found it just led to compiler warnings about unhandled cases in "switch" statements.